src/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <>
+        &copy; 2021 Marni Hager
```

Comment:
Nice use of the copyright symbol. Way to take the time to figure out how to use this.